setup/setup-lokole.sh

```diff
@@ -644,7 +644,19 @@ main() {
   fi
 }
 
-main 2>&1 | tee --append "\${sync_logfile}"
+main_with_retry() {
+  attempts=0
+  timeout=5
+  until "$(attempts -ge 5)"
+  do
+    main 2>&1 | tee --append "\${sync_logfile}" && break
+    attempts="$(attempts+1)"
+    sleep "$(timeout*2)"
```

Comment:
For a true exponential backoff, the timeout should increase with each failed attempt, so what I'd do here is multiply the timeout and then sleep:

```suggestion
    timeout="$((timeout*2))"
    sleep "${timeout}"
```

In this way instead of waiting 5 seconds between each attempt (linear backoff), we'll wait 5, 10, 20, 40, etc. seconds which is exponential. The advantage of using an exponential backoff instead of a linear strategy is that we don't risk creating more failures by overloading the system: if there was a failure, we'll give the system more time to recover before trying again (and given more and more time as the errors accumulate).